File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
+footnoteParagraph.AppendText(" Footnote is modified.")
+'Access paragraph in a Word document.
+paragraph = TryCast(document.Sections(2).Paragraphs(1), WParagraph)
+'Access endnote in the paragraph.
+Dim endnote As WFootnote = TryCast(paragraph.ChildEntities(0), WFootnote)
+'Clear the endnote content.
+endnote.TextBody.ChildEntities.Clear()
+'Add a new paragraph to the body of the endnote.
+Dim endnoteParagraph As WParagraph = TryCast(endnote.TextBody.AddParagraph(), WParagraph)
+'Set the endnote character format.
+endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript
+'Append the endnote text.
+endnoteParagraph.AppendText(" Endnote is modified.")
+'Save a Word document.
+document.Save("Sample.docx", FormatType.Docx)
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.docx"))
+{
+//Load the file stream into a Word document.
+using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+{
+//Access paragraph in Word a document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
+//Access the footnote in the paragraph.
+WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear the footnote content.
+footnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the footnote.
+WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+//Set the footnote character format.
+footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the footnote text.
+footnoteParagraph.AppendText(" Footnote is modified.");
+//Access paragraph a in Word document.
+paragraph = document.Sections[2].Paragraphs[1] as WParagraph;
+//Access the endnote in the paragraph.
+WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
+//Clear the endnote content.
+endnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the endnote.
+WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
+//Set the endnote character format.
+endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the endnote text.
+endnoteParagraph.AppendText(" Endnote is modified.");
+//Save a Word document to MemoryStream.
+MemoryStream stream = new MemoryStream();
+await document.SaveAsync(stream, FormatType.Docx);
+//Save the stream as a Word document file in the local machine.
+Save(stream, "Sample.docx");
+//Please refer to the below link to save the Word document in UWP platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+                }
+            }
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream inputStream = new FileStream(@"../../../Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```

Comment:
@sharonaumao - Please remove the unwanted hint path from here.

Reply:
@Ramaraj-Marimuthu -Done